asciidoc/runtime/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkiverse.qute.web</groupId>
+        <artifactId>quarkus-qute-web-asciidoc-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <artifactId>quarkus-qute-web-asciidoc</artifactId>
+    <name>Quarkus Qute - Asciidoc - Runtime</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-qute</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.yupiik.maven</groupId>
+            <artifactId>asciidoc-java</artifactId>
```

Comment:
I cannot find any info about this library? Is it open source? Where's the source located?

Reply:
@mcruzdev I think that we should at least create a follow-up issue because it might make sense to reuse the infrastructure but provide a different asciidoc library. WDYT?

Reply:
Yeah this is my main question because for sure I think we should have this pure Java option as its way lighter - but it also not able to do all the extensions as one might expect so would be good to know how that would be imagined. Separate impl with different macro or same?